modules/local/select_inflation.nf

```diff
@@ -0,0 +1,46 @@
+process SELECT_INFLATION {
+    tag "Selecting best MCL Inflation"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::bioconductor-r-tidyverse==1.2.1" : null)
+    container "${ workflow.containerEngine == 'docker' ?
+        'austinhpatton123/select_inflation_param:latest': '' }"
```

Comment:
I'm a little confused here - is there not an existing biocontainer with bioconda/bioconductor/tidyverse in it? Or is there some added package in this docker container you made that you need? 
Additionally the docker container should probably be named something a little more informative about what packages are installed in it, although the quay containers sometimes aren't named informative things...

Reply:
Sorry this is just me not deleting the conda specification since I was worried about deleting things for some reason. 

The select_inflation_param docker image includes a couple r packages not included in the tidyverse image, hence my needing to create a custom one. I could probably rename it.